prometheus/doc/watchdog-metrics.md

```diff
@@ -37,47 +37,32 @@ go to k8s container log folder
-| service_current_not_ready_container_count| specific pod contains container status is not ready, value is 1 |
-| service_current_terminated_container_count| specific pod container status is terminated total count, value is 1 |
-| service_current_waiting_container_count| specific pod container status is waiting  total count, value is 1 |
-| service_restarted_container_count| specific pod's container restart total count |
-| pod_current_probe_not_ready| each service occurs readiness probe failed error, condition is not ready, total count |
-| pod_current_phase_failed| each service pods' phase become failed total count |
-| pod_current_phase_unknown| each service pods' phase become unknown total count |
-| container_current_not_ready| each service pods' contains container status is not ready total count |
-| container_current_terminated| each service pods' container status is terminated total count |
-| container_current_waiting | each service pods' container status is waiting  total count |
-| container_accumulation_restart_total| each service pods' container restart total count |
+| pai_pod_count | describe count of pai service like webportal, grafana, lable contains pod status like phase="running", ready="true" |
+| pai_container_count | describe count of pai service container, like pai_pod_count, lable in pai_contain_count contains container status like state="running" |
 
-## K8s Health Metrics
-| Metric name| Labels/tags |
+## Node Health Metrics
+| Metric name| Description |
 | ---------- |  ----------- |
-| apiserver_current_status_error| api server health status, 1 is error |
-| etcd_current_status_error| etcd health status, 1 is error |
-| kubelet_current_status_error| each node kubelet health status, 1 is error |
+| pai_node_count | describe count of node in open pai, lable describe state of node like ready="true" and condition like disk_pressure="false" |
 
-## Node Health Metrics
-| Metric name| Labels/tags |
+## Docker Daemon Health Metrics
+| Metric name| Description |
+| ---------- |  ----------- |
+| docker_daemon_count | has error key in label, if error != "ok", means docker daemon is not functioning correctly |
+
+## K8s Health Metrics
+| Metric name| Description |
 | ---------- |  ----------- |
-| node_current_notready| node status, value 1 is error|
-| notready_node_count| all nodes not ready count|
-| node_current_docker_error| per node docker daemon occurs error, 1 is error |
-| docker_error_node_count| all nodes docker occurs error node count |
+| k8s_api_server_count | has error key in label, if error != "ok", means api server is not functioning correctly |
+| k8s_etcd_count | has error key in label, if error != "ok", means etcd is not functioning correctly |
+| k8s_kubelet_count | has error key in label, if error != "ok", means kubelet is not functioning correctly |
 
+# Alerting
+Alerting rules are under `[prometheus/prometheus-alert](../prometheus-alert)`, we added some basic healthcheck rules for pai service and node.
```

Comment:
suggest we tell end user more details, such as how to add the rule at which file of pai.

Reply:
Added basic info. I intended to rewrite some parts of doc. Put metrics & alerting in one doc file, and add how to emit metric from other pai service.

Reply:
that's useful

Reply:
as your plan, list the rule file path here can make user feel smooth here and minimal cost. otherwise user read alert manager document but can't find the config file

Reply:
yeah, I will submit another doc PR. Let this PR focus on functionality.